kyo-core/shared/src/main/scala/kyo/Clock.scala

```diff
@@ -537,7 +537,7 @@ object Clock:
         frame: Frame,
         reduce: Reducible[Abort[E]]
     ): Fiber[A, reduce.SReduced] < (Sync & S) =
-        Fiber.init {
+        Fiber.initUnscoped {
```

Comment:
I don't think we need to address everything in this PR but, in some cases like this one, it makes more sense to add the pending `Scope`

Reply:
Agreed, I'll take a look through and see if there are some obvious other ones to update. I did that with the other two resource safety PRs.